SageOfCode/Crate
# Style table

What does this PR do?
- Creates a migration to add style_survey to users
- Creates styles table (without relationships at this time)
Where should the reviewer start?
- code/api/src/migrations/user
- code/api/src/migrations/user/styles
- code/api/src/modules/style
How should this be manually tested?
- Testing is not setup at this time for this feature
Any background context you want to provide?
- Ensure that you enter PSQL to drop and recreate Crate and Crate_test databases before running npm run setup:db
What are the relevant tickets?
closes #43 
closes #37 
closes #42
